django/contrib/redirects/migrations/0004_remove_site.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('redirects', '0003_migrate_sites'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
```

Comment:
This is a long list of migrations. I would recommend to try to get as many operations into one migration as you can. Loading a lot of migrations will slow down Django during startup and all migration commands.

Reply:
I'd also prefer it, but data migration needs to be separated.